### PR TITLE
Editor: Handle LN placement when StartTime > EndTime

### PR DESCRIPTION
**Pull request by staravia**
Solves https://github.com/Quaver/Quaver/issues/734

### CODE REVIEW

**Comment by YaLTeR** [comment]
You didn't revert the submodule properly.

**Comment by staravia** [comment]
> You didn't revert the submodule properly.

resolved

File: Quaver.Shared/Database/Maps/OtherGameMapDatabaseCache.cs [hunk 1]
```diff
@@ -199,7 +199,7 @@ private static List<OtherGameMap> LoadOsuBeatmapDatabase()
                     var newMap = new OtherGameMap()
                     {
                         Md5Checksum = map.BeatmapChecksum,
-                        Directory = map.FolderName.Trim(),
+                        Directory = map.FolderName,
```
Comment (YaLTeR) [inline Quaver.Shared/Database/Maps/OtherGameMapDatabaseCache.cs]: You merged the wrong commit because this isn't supposed to disappear

Comment (staravia) [reply Quaver.Shared/Database/Maps/OtherGameMapDatabaseCache.cs]: wtf

**Pull request closed without merging**